ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -332,6 +340,13 @@ open class NavigationBar: UINavigationBar, TwoLineTitleViewDelegate {
     private var navigationBarShadowObserver: NSKeyValueObservation?
     private var usesLargeTitleObserver: NSKeyValueObservation?
 
+    private var backButtonItem: UIBarButtonItem?
```

Comment:
Looks like this is never set outside of init. Should this be a `let`, and non-optional?

Reply:
Will fix.